lib/config/options/index.ts

```diff
@@ -1499,6 +1499,38 @@ const options: RenovateOptions[] = [
     env: false,
     supportedPlatforms: ['github'],
   },
+  {
+    name: 'osvVulnerabilityAlerts',
+    description: 'Use vulnerability alerts from osv.dev',
+    type: 'boolean',
+    default: false,
+    releaseStatus: 'unpublished',
+    // GitHub is not supported as it already provides vulnerability alerts
+    supportedPlatforms: [
+      'azure',
+      'bitbucket',
+      'bitbucket-server',
+      'gitea',
+      'gitlab',
+    ],
+    supportedManagers: [
```

Comment:
Why is support limited by manager? A security update is not anything different from a normal update, I'd say. If there is a maven dependency that's managed via the Regex manager, shouldn't that just work?

If anything, it may be restricted to data sources, but that's only so because [osv.dev](https://osv.dev/) does not have data for all of them.

Reply:
I've updated this to map datasource -> OSV ecosystem. Thanks!